<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Gui::Pref::TGeneral</class>
 <widget class="QWidget" name="Gui::Pref::TGeneral">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>491</width>
    <height>554</height>
   </rect>
  </property>
  <layout class="QVBoxLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QTabWidget" name="general_tab_widget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="general_tab">
      <attribute name="title">
       <string>&amp;General</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="playerbin_layout">
         <item>
          <widget class="QLabel" name="playerbin_label">
           <property name="text">
            <string notr="true">Player &amp;executable:</string>
           </property>
           <property name="wordWrap">
            <bool>false</bool>
           </property>
           <property name="buddy">
            <cstring>playerbin_edit</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="FileChooser" name="playerbin_edit"/>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QGroupBox" name="media_settings_group">
         <property name="title">
          <string>Media settings</string>
         </property>
         <layout class="QVBoxLayout">
          <item>
           <widget class="QCheckBox" name="remember_all_check">
            <property name="text">
             <string>&amp;Remember settings for every file (audio track, subtitles...)</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QGridLayout">
            <item row="0" column="0" rowspan="3">
             <spacer>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Preferred</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>61</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="0" column="1">
             <widget class="QCheckBox" name="remember_time_check">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="sizePolicy">
               <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>Remember &amp;time position for every file</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <layout class="QHBoxLayout">
              <item>
               <widget class="QLabel" name="filesettings_method_label">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="text">
                 <string>&amp;Store settings in</string>
                </property>
                <property name="buddy">
                 <cstring>filesettings_method_combo</cstring>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="filesettings_method_combo">
                <property name="enabled">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
              <item>
               <spacer>
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
            <item row="1" column="1">
             <widget class="QCheckBox" name="remember_volume_check">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Remember &amp;volume for every file</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="screenshots_group">
         <property name="title">
          <string>Screenshots</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QCheckBox" name="use_screenshots_check">
            <property name="text">
             <string>&amp;Enable screenshots</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <spacer name="horizontalSpacer_3">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Preferred</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QLabel" name="screenshots_dir_label">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>&amp;Folder:</string>
              </property>
              <property name="wordWrap">
               <bool>false</bool>
              </property>
              <property name="buddy">
               <cstring>screenshot_edit</cstring>
              </property>
             </widget>
            </item>
            <item>
             <widget class="FileChooser" name="screenshot_edit">
              <property name="enabled">
               <bool>false</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <spacer name="horizontalSpacer_4">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Preferred</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QLabel" name="screenshot_template_label">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Temp&amp;late:</string>
              </property>
              <property name="buddy">
               <cstring>screenshot_template_edit</cstring>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="screenshot_template_edit">
              <property name="enabled">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="screenshot_format_label">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>F&amp;ormat:</string>
              </property>
              <property name="buddy">
               <cstring>screenshot_format_combo</cstring>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="screenshot_format_combo">
              <property name="enabled">
               <bool>false</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="Line" name="line_7">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="pause_if_hidden_check">
         <property name="text">
          <string>Pause &amp;when minimized</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="close_on_finish_check">
         <property name="text">
          <string>&amp;Close when playback finished</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>FileChooser</class>
   <extends>QLineEdit</extends>
   <header>filechooser.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>general_tab_widget</tabstop>
  <tabstop>playerbin_edit</tabstop>
  <tabstop>remember_all_check</tabstop>
  <tabstop>remember_time_check</tabstop>
  <tabstop>remember_volume_check</tabstop>
  <tabstop>filesettings_method_combo</tabstop>
  <tabstop>use_screenshots_check</tabstop>
  <tabstop>screenshot_edit</tabstop>
  <tabstop>screenshot_template_edit</tabstop>
  <tabstop>screenshot_format_combo</tabstop>
  <tabstop>pause_if_hidden_check</tabstop>
  <tabstop>close_on_finish_check</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>remember_all_check</sender>
   <signal>toggled(bool)</signal>
   <receiver>filesettings_method_label</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>229</x>
     <y>233</y>
    </hint>
    <hint type="destinationlabel">
     <x>93</x>
     <y>298</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>remember_all_check</sender>
   <signal>toggled(bool)</signal>
   <receiver>remember_time_check</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>280</x>
     <y>323</y>
    </hint>
    <hint type="destinationlabel">
     <x>293</x>
     <y>355</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>use_screenshots_check</sender>
   <signal>toggled(bool)</signal>
   <receiver>screenshot_format_label</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>249</x>
     <y>243</y>
    </hint>
    <hint type="destinationlabel">
     <x>359</x>
     <y>310</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>use_screenshots_check</sender>
   <signal>toggled(bool)</signal>
   <receiver>screenshot_format_combo</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>249</x>
     <y>243</y>
    </hint>
    <hint type="destinationlabel">
     <x>430</x>
     <y>310</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>use_screenshots_check</sender>
   <signal>toggled(bool)</signal>
   <receiver>screenshot_template_label</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>249</x>
     <y>243</y>
    </hint>
    <hint type="destinationlabel">
     <x>61</x>
     <y>307</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>use_screenshots_check</sender>
   <signal>toggled(bool)</signal>
   <receiver>screenshot_edit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>229</x>
     <y>256</y>
    </hint>
    <hint type="destinationlabel">
     <x>254</x>
     <y>288</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>use_screenshots_check</sender>
   <signal>toggled(bool)</signal>
   <receiver>screenshot_template_edit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>249</x>
     <y>243</y>
    </hint>
    <hint type="destinationlabel">
     <x>285</x>
     <y>307</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>remember_all_check</sender>
   <signal>toggled(bool)</signal>
   <receiver>filesettings_method_combo</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>229</x>
     <y>233</y>
    </hint>
    <hint type="destinationlabel">
     <x>181</x>
     <y>298</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>use_screenshots_check</sender>
   <signal>toggled(bool)</signal>
   <receiver>screenshots_dir_label</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>229</x>
     <y>256</y>
    </hint>
    <hint type="destinationlabel">
     <x>43</x>
     <y>288</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>remember_all_check</sender>
   <signal>toggled(bool)</signal>
   <receiver>remember_volume_check</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>245</x>
     <y>93</y>
    </hint>
    <hint type="destinationlabel">
     <x>257</x>
     <y>144</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
